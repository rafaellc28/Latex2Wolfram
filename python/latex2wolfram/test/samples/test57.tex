% Test logarithm with base expression
\log_{11}(1.2)
