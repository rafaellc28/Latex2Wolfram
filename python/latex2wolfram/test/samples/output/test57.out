log11(1.2)
